sdk/informaticadatamanagement/Azure.ResourceManager.InformaticaDataManagement/samples/Generated/Samples/Sample_InformaticaOrganizationCollection.cs

```diff
@@ -22,7 +22,7 @@ public partial class Sample_InformaticaOrganizationCollection
         public async Task CreateOrUpdate_OrganizationsCreateOrUpdate()
         {
             // Generated from example definition: 2024-05-08/Organizations_CreateOrUpdate_MaximumSet_Gen.json
-            // this example is just showing the usage of "InformaticaOrganizationResource_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Organizations_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `creaetOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/informatica/Informatica.DataManagement.Management/main.tsp#L62

Reply:
renaming `Organization` to `InformationOrganization` in https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/informatica/Informatica.DataManagement.Management/client.tsp#L11